runfrontendtests.sh

```diff
@@ -6,5 +6,5 @@ python -m tests.e2e.frontend_tests_initdb
 python runcypressserver.py &
 SERVER_PID=$!
 cd funnel/assets
-npx cypress run --browser chrome --record --key $RECORD_KEY
+npx cypress run --browser chrome
```

Comment:
Cypress limit for this month has exceeded so removed the record key to run tests. Will add  it back before merging.